docs/admin/deployment.md

```diff
@@ -485,14 +485,19 @@ storage selection.
 
 #### Discovery
 
-`dmg storage scan` can be run to query remote running `daos_server`
-processes over the management network.
-
-`daos_server storage scan` can be used to query local `daos_server` instances
-directly (scans locally-attached SSDs and Intel Persistent Memory Modules usable
-by DAOS).
-NVMe SSDs need to be made accessible first by running
-`daos_server storage prepare --nvme-only`.
+DAOS tools will discover NVMe SSDs and Persistent Memory Modules using the storage scan commands.
+
+`dmg storage scan` can be run to query remote running `daos_server` processes over the management
+network.
+
+`daos_server storage scan` can be used to query storage on local hosts directly.
+
+NVMe SSDs need to be made accessible first by running `daos_server storage prepare --nvme-only`.
+The default way for DAOS to access NVMe storage is through SPDK via the VFIO user-space driver.
+To use an alternative driver with SPDK, set `--disable-vfio` in the storage prepare command to
+fallback to using UIO user-space driver with SPDK instead. The option is only effective when
+IOMMU and VFIO are enabled on the host.
```

Comment:
A little confusing in the phrasing. Might be better to put it another way, something like: "If IOMMU and VFIO are not enabled in the BIOS, the alternative driver will be used by default." At least that is what I am taking this to mean.

Reply:
done